config/ar-lib

```diff
@@ -0,0 +1,270 @@
+#! /bin/sh
+# Wrapper for Microsoft lib.exe
```

Comment:
Where does this file come from (ar-lib)? Can it be autogenerated by autotools?

Reply:
This is my bad. it's generated by the autotools and I thought I had removed it from the repo already